pytket/pytket/circuit/logic_exp.py

```diff
@@ -763,7 +767,9 @@ def create_logic_exp(op: Ops, args: Sequence[ArgType]) -> LogicExp:
     return create_reg_logic_exp(op, reg_args)
 
 
-def create_predicate_exp(op: Ops, args: Sequence[ArgType]) -> PredicateExp:  # noqa: PLR0911
+def create_predicate_exp(  # noqa: PLR0911
+    op: Ops, args: Sequence[ArgType]
```

Comment:
I think these styling changes messesed with the position of the `# noqa` comments causing the linting checks to fail.